components/SettingsModal.vue

```diff
@@ -367,8 +343,6 @@ const selected = ref(true); // true = general, false = calendar
 // General Settings
 const showWeekend = ref(true);
 const startWeekOn = ref("Monday");
-const dateFormat = ref("DD-MM-YY");
-const timeFormat = ref("12-Hour-Time");
 // const currentTheme = ref("dark");
```

Comment:
commented code

Reply:
Resolved in b5a7640d3df38d757f8daf756898bf742f468212